app/templates/index.html

```diff
@@ -1,40 +1,75 @@
 {% extends "base.html" %}
-{% block title %}Index{% endblock %}
+{% block title %}Match mentees to mentors{% endblock %}
 {% block head %}
     {{ super() }}
 {% endblock %}
 {% block content %}
 
-    <h1 class="home--heading">Mentor matcher</h1>
-    <p class="lead">Welcome to the mentor matcher.</p>
+<div class="grid-row">
+  <header class="page--header">
+	  <div class="grid-column-full">
+			<span class="page--caption">Tools</span>
+		  <h1 class="page--heading">Mentor matcher</h1>
+	  </div>
+	  <div class="grid-column-two-thirds">
+		  <p class="page--excerpt">This tool helps you to match mentors to mentees as part of the Civil Service LGBT+ mentoring programme.</p>
+	  </div>
+  </header>
+</div>
 
-	<hr>
+<div class="grid-row">
+  <div class="grid-column-two-thirds">
+		<details>
+			<summary>About this tool</summary>
+				<p>Explanatory text about the took can go here...</p>
+		</details>
+  </div>
+</div>
 
-    <div>
-	    <h2>Tasks</h2>
-	    <p>Pick a task length.</p>
-	    <div class="button-group" role="group" aria-label="Basic example">
-			<button type="button" class="button" onclick="handleClick('small')">50 participants</button>
-			<button type="button" class="button" onclick="handleClick('large')">500 participants</button>
-	    </div>
-	</div>
+<div class="grid-row main-wrapper--lg">
+  <div class="grid-column-full">
+    <h2>Tasks</h2>
+    <div class="page--content">
+      <p>Pick a task length.</p>
+      <div class="button-group" role="group" aria-label="Basic example">
+		  <button type="button" class="button" onclick="handleClick('small')">50 participants</button>
+		  <button type="button" class="button" onclick="handleClick('large')">500 participants</button>
+      </div>
+    </div>
+  </div>
+</div>
 
-	<hr>
+<dialog class="dialog" id="dialog-box">
+  <!-- this dialog will only show when required -->
+  <!-- note that this won't work in Safari yet -->
+  <section class="dialog--header"><h2 class="heading-lg dialog--title">Matching complete</h2></section>
+	<section class="dialog--content">
+    <p>The matching process is complete.</p>
+    <p>Use the <kbd>Download matches</kbd> button to get the results.</p>
+	</section>
+	<footer class="dialog--actions">
+		<a href="" class="button button--dialog" id="download-matches" title="Download matching results" download>Download matches</a>
+		<button onclick="closeDialog('dialog-box')" class="button button--secondary button--dialog button--close">Close this dialog</button>
+	</footer>
+</dialog>
 
-	<div>
-		<h2>Task Status</h2>
-
-		<table class="table">
-			<thead>
-				<tr>
-					<th>ID</th>
-					<th>Status</th>
-					<th>Result</th>
-				</tr>
-			</thead>
-			<tbody id="tasks">
-
-			</tbody>
-		</table>
-	</div>
+<div class="grid-row main-wrapper--lg">
```

Comment:
I want to be fairly aggressive with removing this - I wonder if we can cut it all the way down to just the `status`? I'm sure users don't get any value from the id of the celery job!

Reply:
I have left this in for now in case we wanted to recycle it for a different purpose, but we could remove it if you think it won't be needed.

Shall we keep it for now and then aim to remove on the next pass through? At least it can be ignored in its current form!

Reply:
Yeah, okay. Leave it for now and we'll do another tidy on the next pass. Next feature I think will be the 'processing' journey, so that might be the time to work out what we actually want users to see